### PR TITLE
Added policy tables

### PR DESCRIPTION
**Pull request by aburdenthehand**
@vikram-redhat 
PRx19 - You know the drill...

### CODE REVIEW

**Comment by aburdenthehand** [comment]
@ahardin-rh - Does that look about right? (Sorry about this PR nonsense. Some fiasco on my end)

File: architecture/additional_concepts/authorization.adoc [hunk 1]
```diff
@@ -228,17 +228,412 @@ TIP: Remember that xref:authentication.adoc#users-and-groups[users
...
-Cluster administrators can visualize these roles, including a matrix of the
-verbs and resources each are associated using the CLI to
+Cluster administrators can investigate policy roles by using the `oc describe` command, which displays a matrix of the `Verbs`, `Non-Resource URLs`, `Extension`, `Resource Names`,`API Groups`, and `Resources` that apply to the policy role.
+
+----
+$ oc describe clusterrole <role>
+----
+
+The rule verbs apply to the objects listed in that row. For example, if the `create` verb is applied to the `serviceaccounts` resource, the holder is allowed to create service accounts but no other resource unless otherwise specified in the role matrix. +
+The verbs are described in the following table.
```
Comment (ahardin-rh) [inline architecture/additional_concepts/authorization.adoc]: There still looks to be a rogue + here. Does it build okay when you run [`asciibinder build`](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/tools_and_setup.adoc#install-asciibinder-and-dependencies) ?

Comment (vikram-redhat) [reply architecture/additional_concepts/authorization.adoc]: @aburdenthehand ^^

Comment (aburdenthehand) [reply architecture/additional_concepts/authorization.adoc]: Yeah, that's intentional. That sentence doesn't warrant a para to itself but should be distinct from the previous.

**Comment by vikram-redhat** [comment]
@nengard - please peer review.

**Comment by vikram-redhat** [comment]
@aburdenthehand - will this apply to 3.6 onwards or 3.4? And in Dedicated or Online docs?

**Comment by vikram-redhat** [comment]
@aburdenthehand - I am going to go ahead and merge and pick it to 3.6 as this needs to go out today. Let me know about the branches when you can. @nengard - if you have any review comments on it, we can incorporate them in the next iteration.

**Comment by vikram-redhat** [comment]
Hey @nengard  - did you have any review comments on this?

**Comment by vikram-redhat** [comment]
Confirmed with Andrew that it applies to 3.5 as well. Adding that in.

**Pull request merged**